MyBlog.Infrastructure/Entities/User.cs

```diff
@@ -0,0 +1,22 @@
+﻿using MyBlog.Infrastructure.Entities.Enum;
+
+namespace MyBlog.Infrastructure.Entities;
+
+public class User
+{
+    public int Id { get; set; }
+
+    public string FirstName { get; set; }
+
+    public string LastName { get; set; }
+
+    public string Password { get; set; }
+
+    public string Email { get; set; }
+
+    public Role Role { get; set; }
+
+    public DateTime DateCreateAccount { get; set; }
```

Comment:
init this property by default:
public DateTime DateCreateAccount { get; set; } = DateTime.Now